### DISCUSSION TITLE
Update Sentry integration

### DISCUSSION THREAD

**Discussion by uhlhosting** [postalserver/postal#2406] [Feature suggestions]
According to : https://docs.sentry.io/platforms/ruby/migration/ the ruby 2.x is not supported anymore, and they also migrated from raven-ruby to sentry-ruby SDK . The docs explain all changes.

**Comment by willpower232** [comment]
Good to know, probably worth being a PR if you have the time and an issue if you don't :pray:

**Reply by uhlhosting** [reply]
@willpower232 will have to stay as a issue for now!

**Reply by asplinsol** [reply]
FYI the migration to sentry-ruby has been done via PR #2392 - think this can be closed?
